### DISCUSSION TITLE
Installation options should be remembered when upgrading

### DISCUSSION THREAD

**Discussion by FCrane** [rustdesk/rustdesk#4361] [Feature Request]
### Bug Description

Hi!

When upgrading Rustdeks (e.g. installing a new nightly by renaming it to "rustdesk-install.exe", the options are not remembered, which is quite annoying. The "Installation path", "Create start menu shortcut", "Create desktop icon" and "Install virtual display driver" should be remembered from the last installation for quicker, more consistent and convenient upgrading.

Regards!

### How to Reproduce

Upgrade Rustdesk

### Expected Behavior

All options should be remembered from previous installation

### Operating system(s) on local side and remote side

Windows 11 x64

### RustDesk Version(s) on local side and remote side

Latest nightly

### Screenshots

n/a

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/4331

**Comment by rustdesk** [comment]
@21pages follow up this, but lowest priority, let's do the other jobs first. No need to add more options for remembering these, we can check if current installation have these features. e.g. check if the desktop shortcut file exists. Also please confirm if update_me function does not override these options,https://github.com/rustdesk/rustdesk/blob/c1b2b55c530cf1f138610c3d8f998e5555978352/src/platform/windows.rs#L939

**Comment by rustdesk** [comment]
Contribution is welcome, this is an easy job.

Write some code in Rust or dart to check if  the desktop shortcut file / startmeu file exists, here are the paths.
https://github.com/rustdesk/rustdesk/blob/c1b2b55c530cf1f138610c3d8f998e5555978352/src/platform/windows.rs#L1274

then modify here.
https://github.com/rustdesk/rustdesk/blob/c1b2b55c530cf1f138610c3d8f998e5555978352/flutter/lib/desktop/pages/install_page.dart#L152
